libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -127,6 +127,19 @@ class MapboxTripNotification constructor(
         currentInstructionText = null
         currentDistanceText = null
 
+        collapsedNotificationRemoteViews?.apply {
+            setTextViewText(R.id.notificationDistanceText, "")
+            setTextViewText(R.id.notificationArrivalText, "")
+            setTextViewText(R.id.notificationInstructionText, "")
+        }
```

Comment:
Let's remove the collapsed notification view altogether as @noracalabrese suggested